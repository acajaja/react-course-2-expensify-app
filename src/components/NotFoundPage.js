import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
    <div>
        <p>
            Ooops! 404 :(
            <br></br>
            <Link to="/" title="Home">Home</Link>
        </p>
    </div>
);

export default NotFoundPage;
