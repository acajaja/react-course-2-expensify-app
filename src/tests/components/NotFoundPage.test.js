import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

test('Test the 404 page', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});
