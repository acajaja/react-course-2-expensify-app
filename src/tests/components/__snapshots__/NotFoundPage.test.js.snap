// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the 404 page 1`] = `
<div>
  <p>
    Ooops! 404 :(
    <br />
    <Link
      replace={false}
      title="Home"
      to="/"
    >
      Home
    </Link>
  </p>
</div>
`;
